content/en/docs/languages/java/instrumentation.md

```diff
@@ -676,6 +676,36 @@ Tracer tracer = GlobalOpenTelemetry.getTracer("instrumentation-scope-name", "ins
 Note that you can't force end users to configure the global, so this is the most
 brittle option for library instrumentation.
 
+### Acquiring a Tracer in Java agent
```

Comment:
```suggestion
### Acquiring a tracer in Java agent
```

Sentence case in titles unless there's a proper noun, such as "Spring Boot".

Reply:
also changed the existing headers